zilliqa/src/message.rs

```diff
@@ -77,7 +85,7 @@ impl Vote {
     ) -> Self {
         let mut bytes = Vec::new();
         bytes.extend_from_slice(block_hash.as_bytes());
-        bytes.extend_from_slice(&view.to_be_bytes());
+        //bytes.extend_from_slice(&view.to_be_bytes());
```

Comment:
the votes and the QC are signing different things (not both include view) - address in a split-off PR